src/core/src/pass/smart_reshape/strided_slice_squeeze.cpp

```diff
@@ -57,13 +56,35 @@ ngraph::pass::StridedSliceSqueeze::StridedSliceSqueeze() {
             }))
             return false;
 
+        auto const_axes_tmp = const_axes->cast_vector<int64_t>();
+
         const auto& axes = normalize_axes(squeeze->description(),
-                                          const_axes->cast_vector<int64_t>(),
-                                          squeeze->get_input_partial_shape(0).rank());
+                                          const_axes->cast_vector<int64_t>(),           // [1]
```

Comment:
comments needs to make sense

Reply:
thanks, they are legacy of my debugging... removed.